.github/workflows/build-book.yaml

```diff
@@ -76,6 +64,12 @@ jobs:
           git config --local user.name "$GITHUB_ACTOR"
           git config --local user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Cache bookdown results
+        uses: actions/cache@v2
+        with:
+          path: _bookdown_files
+          key: bookdown-8-${{ hashFiles('**/*Rmd') }}
```

Comment:
I removed the restore key here — I think that may have been exacerbating the caching issue.